unskript-ctl/unskript_ctl_notification.py

```diff
@@ -194,10 +194,10 @@ def create_temp_files_of_failed_check_results(self,
         return list_of_failed_files
 
     def create_script_summary_message(self, output_metadata_file: str):
-        message = ''
+        # message = ''
         if os.path.exists(output_metadata_file) is False:
             self.logger.error(f"ERROR: The metadata file is missing, please check if file exists? {output_metadata_file}")
-            return message
+            return ''
```

Comment:
Not sure if we need this change. Do you want to remove the variable `message` altogether?